### DISCUSSION TITLE
Re-order sections, change player layout and slider for time elapsed

### DISCUSSION THREAD

**Discussion by YvesZH** [punxaphil/custom-sonos-card#48] [Ideas]
I would also like to thank you for the great card.  I mainly use lovelace on my mobile.  is it possible to arrange the order differently (first the speaker selection, then the player)?  and as an additional idea: i would like the space for the album picture to be smaller.  The control buttons (play...) are larger and only selected buttons (no volume and shuffle buttons).  And a slider where you could jump back and forth in the audio file would also be great.  Thank you again 👍

**Comment by punxaphil** [comment]
Interesting! Some sketches or wireframes would be nice, to make sure I understand.

**Comment by punxaphil** [comment]
Pretty everything is stylable in this card now, so you can probably solve this with some CSS skills.
https://github.com/johanfrick/custom-sonos-card#css-styling
